usr/lib/cca_stdll/cca_specific.c

```diff
@@ -5046,9 +5056,13 @@ CK_RV token_specific_aes_ecb(STDLL_TokData_t * tokdata,
     unsigned char exit_data[1];
     CK_BYTE *local_out = out_data;
     CK_ATTRIBUTE *attr = NULL;
-    CK_MECHANISM mech = { CKM_AES_ECB, NULL, 0 };
     long int key_len;
     CK_RV rc;
+#ifndef NO_PKEY
+    CK_MECHANISM mech = { CKM_AES_ECB, NULL, 0 };
+#else
+    UNUSED(session);
```

Comment:
hm, I would make 2 #ifdef blocks here. One for variable declaration, and one for the UNUSED. 
UNUSED really does not belong to variable declaration, and hsould have ablank line between variaböles and UNUSED.

Same for the other functions below.

Reply:
Done.